create_coda_files/create_coda_files.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import os
+import time
+import random
+
+import pytz
+from core_data_modules.cleaners import somali
+from core_data_modules.traced_data import Metadata
+from core_data_modules.traced_data.io import TracedDataJsonIO, TracedDataCodaIO
+from core_data_modules.util import IOUtils
+from dateutil.parser import isoparse
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Takes a TracedData json file and converts it to a CODA file")
+    parser.add_argument("user", help="User launching this program")
+    parser.add_argument("json_input_path", metavar="json-input-path",
+                        help="Path to the input JSON file, containing a list of serialized TracedData objects")
+    parser.add_argument("flow_name", metavar="flow-name",
+                        help="Name of activation flow from which this data was derived")
+    parser.add_argument("variable_name", metavar="variable-name",
+                        help="Name of message variable in flow")
+    parser.add_argument("coda_output_path", metavar="coda-output-path",
+                        help="Path to a Coda file to write processed messages to")
+    parser.add_argument("--prev_coda_path", metavar="prev-coda-path",
+                        help="Path to a Coda file containing previously coded messages")
+
+    args = parser.parse_args()
+    user = args.user
+    json_input_path = args.json_input_path
+    flow_name = args.flow_name
+    variable_name = args.variable_name
+    coda_output_path = args.coda_output_path
+    if args.prev_coda_path:
+        prev_coda_path = args.prev_coda_path
+    else:
+        prev_coda_path = None
+    
+    # Load data from JSON file
+    with open(json_input_path, "r") as f:
+        show_messages = TracedDataJsonIO.import_json_to_traced_data_iterable(f)
+
+    # Filter out test messages sent by AVF.
+    show_messages = [td for td in show_messages if not td.get("test_run", False)]
```

Comment:
Don't you want this to be commented out (with a TODO here and an issue against the repo) until you finish testing?